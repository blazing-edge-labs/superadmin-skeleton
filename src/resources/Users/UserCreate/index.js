import React from 'react'
import {
  Create,
  SimpleForm,
  TextInput,
} from 'admin-on-rest'

const UserCreate = (props) => (
  <Create {...props}>
    <SimpleForm>
      <TextInput source="email" type="email"/>
      <TextInput source="password" type="password" default=""/>
    </SimpleForm>
  </Create>
)

export default UserCreate
