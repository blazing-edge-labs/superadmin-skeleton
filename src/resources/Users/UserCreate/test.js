import React from 'react'
import { shallow } from 'enzyme'

import UserCreate from './'

describe('UserCreate', () => {
  const userCreate = shallow(<UserCreate/>)

  it('renders properly', () => {
    expect(userCreate).toMatchSnapshot()
  })
})
