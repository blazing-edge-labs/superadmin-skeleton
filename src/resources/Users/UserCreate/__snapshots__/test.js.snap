// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserCreate renders properly 1`] = `
<Connect(getContext(Create))>
  <Connect(ReduxForm)>
    <TextInput
      addField={true}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      options={Object {}}
      source="email"
      type="email"
    />
    <TextInput
      addField={true}
      default=""
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      options={Object {}}
      source="password"
      type="password"
    />
  </Connect(ReduxForm)>
</Connect(getContext(Create))>
`;
